### ISSUE TITLE
Reset password bug

### ISSUE THREAD

**Issue by afraarslan** [bounswe/bounswe2020group8#239]
**Describe the bug**
Since the backend team made some changes in the reset password endpoint, the request in the reset always gives an error. 
`400-bad-request`

**Possible code location**
It is in the `~/src/components/Login/ResetPassword.js` file. (In the post request)

**Expected behavior**
It should check the new password whether it is strong or not and then it sends a post request with the correct parameters.

**Screenshots**
![image](https://user-images.githubusercontent.com/32341138/101981240-731fb080-3c7c-11eb-9134-b60905d8ec7e.png)

**Additional context**
Since password check is also required, I will fix both of them together.

**Comment by afraarslan** [comment]
I updated the post request and fixed the bug. It is in the `profile` branch. It will be used after #252 merged.

**Comment by afraarslan** [comment]
This bug is fixed and merge into the web branch with #252.
